pages/embedded-portal.tsx

```diff
@@ -26,20 +26,33 @@ import StepList, { Step } from "../components/shared/step-list";
 import Workspace from "../components/embedded-portal/workspace";
 import { theme } from "../lib/theme";
 import { useFlashMessages, useOnClickOutside } from "../lib/hooks/usehooks";
+import { Flatfile } from "@flatfile/api";
+import { WorkflowType, getSpace, getWorkbook } from "../lib/flatfile";
+
+type WorkbookConfig = {
+  name: string;
+  labels?: string[];
+  spaceId: Flatfile.SpaceId;
+  environmentId: Flatfile.EnvironmentId;
+  /** Sheets must be <= 50 */
+  sheets?: Flatfile.SheetConfig[];
+  actions?: Flatfile.Action[];
+};
 
 interface Props {
   accessToken: string;
   environmentToken: string;
   lastSyncedAt?: string;
   existingSpace: Space;
+  workbookConfig: Pick<WorkbookConfig, "name" | "sheets" | "actions">;
```

Comment:
I think there's a type from the package for this, it was like CreateWorkbookConfig or something?